src/simple_commander/main.py

```diff
@@ -449,6 +449,7 @@ def set_invaders(self, count):
     def get_units(self):
         units = []
         if len(self.units):
+            #TODO: limit units to top 10
```

Comment:
On UI we will see only top 10 or 15 users with the highest scores. So no reason to return information about all users